built_value_generator/test/enum_class_generator_test.dart

```diff
@@ -222,7 +222,7 @@ class TestEnum extends EnumClass {
 }
 '''), endsWith(r'''Please make the following changes to use EnumClass:
 
-1. Have exactly one constructor: const TestEnum._(String name) : super(name);'''));
+1. Have exactly one constructor: const TestEnum._((final )?String name) : super(name);'''));
```

Comment:
Could you please tweak to keep the original failure message?

If people want "final" they will presumably try it without being asked :) but otherwise I think it's less confusing to highlight just the one way forward.

Reply:
Makees sense, updated :)